shared/components/NextCommunityEvent/index.js

```diff
@@ -15,6 +17,24 @@ function googleMapsUrl(location) {
   return `http://www.google.com/maps/place/${latitude},${longitude}`;
 }
 
+function eventLocation(location) {
+  return pathOr(placeholderText, ['address'], location);
+}
+
+function eventDate(date) {
+  if (date && date.iso) {
+    return formatDate(date, 'dddd, Do MMMM YYYY');
+  }
+  return placeholderText;
+}
+
+function eventTime(startDateTime, endDateTime) {
```

Comment:
They're covered by the component tests. Basically just extracted them because it was getting hard to write it all inline.